cucumber/authenticators_jwt/features/authn_jwt_fetch_signing_key.feature

```diff
@@ -569,3 +569,53 @@ Feature: JWT Authenticator - Fetch signing key
     """
     cucumber:host:myapp successfully authenticated with authenticator authn-jwt service cucumber:webservice:conjur/authn-jwt/raw
     """
+
+  Scenario: provider-uri TLS certificate validation
+    Given I load a policy:
+    """
+    - !policy
+      id: conjur/authn-jwt/keycloak
+      body:
+      - !webservice
+
+      - !variable
+        id: provider-uri
+    """
+    And I am the super-user
+    And I successfully set authn-jwt "provider-uri" variable in service "keycloak" with untrusted TLS server
+    And I fetch an ID Token for username "alice" and password "alice"
+    And I save my place in the log file
+    When I authenticate via authn-jwt with the ID token
+    Then the HTTP response status code is 502
+    And The following appears in the log after my savepoint:
+    """
+    CONJ00011E Failed to discover Identity Provider (Provider URI: 'https://jwks'). Reason: '#<OpenIDConnect::Discovery::DiscoveryFailed: SSL_connect returned=1 errno=0 state=error: certificate verify failed (self signed certificate)>
+    """
+
+  Scenario: jwks-uri TLS certificate validation
+    Given I load a policy:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !webservice
+
+      - !variable
+        id: jwks-uri
+    """
+    And I am the super-user
+    And I successfully set authn-jwt "jwks-uri" variable in service "raw" with untrusted TLS server
```

Comment:
The same as above. I think that the sentence should be simpler means to have an explicit value in it.